hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMiniOzoneCluster.java

```diff
@@ -58,14 +58,12 @@
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.yaml.snakeyaml.Yaml;
 
 /**
  * Test cases for mini ozone cluster.
  */
-@Ignore
```

Comment:
Why was it ignored and how is the problem fixed by this patch? Do we need to run multiple checks to prove that there is no intermittency any more?

Reply:
[Disabled on Dec 27](https://github.com/apache/hadoop-ozone/commit/9000323e3741a39cc8b645a3cb2acad7f13d2a88) on the `integration-test-cleanup` branch due to [a failed run](https://github.com/apache/hadoop-ozone/runs/365409169).  Unfortunately logs are no longer available to check what was the problem.

Reply:
Thanks to check it. It can be because a flakiness. In this case it would be great to have at least 3-5 green builds before the merge .

Reply:
> It can be because a flakiness. In this case it would be great to have at least 3-5 green builds before the merge .

Running 20x, passed 5x so far:
https://github.com/adoroszlai/hadoop-ozone/runs/570025575

Reply:
20/20 passed

Reply:
This works, so leaving this.